srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -272,6 +273,27 @@ private void retrieveHealthFactors(final int dfn, final Patient patient)
         }
     }
     
+    private void retrieveActiveMedications(final int dfn, final Patient patient)
+    {
+        try
+        {
+            patient.getActiveMedications().clear();
+            // Leave the start and end dates blank so that we get only the current medications.
+            final List<String> rpcResults = fProcedureCaller.doRpc(
+                    fDuz, RemoteProcedure.GET_ACTIVE_MEDICATIONS, String.valueOf(dfn), "", "");
+            for(final String medResult: rpcResults)
+            {
+                final List<String> medInfo = Splitter.on('^').splitToList(medResult);
+                patient.getActiveMedications().add(medInfo.get(1));
+            }
+            LOGGER.debug("Retrieved Active Medications: {} ", patient.getActiveMedications());
+        }
+        catch(final Exception e)
+        {
+            LOGGER.warn("Unable to retrieve active medications. {}", e.toString());
```

Comment:
Nooo, this eliminates display of the stracktrace. Remove the placeholder and just pass `e` as the second parameter.